src/ui/win32/bindings/ControlBinding.hh

```diff
@@ -5,6 +5,8 @@
 #include "ui\BindingBase.hh"
 #include "ui\win32\DialogBase.hh"
 
+#include "ui\viewmodels\MessageBoxViewModel.hh"
```

Comment:
Why was this moved? Please keep unnecessary headers out of header files as they have to be parsed for every file that includes the container header file.